contracts/Oracle/Calculations/Sushiswap.sol

```diff
@@ -38,34 +40,28 @@ interface IERC20 {
     function balanceOf(address account) external view returns (uint256);
 }
 
-contract CalculationsSushiswap {
-    address public primaryRouterAddress;
-    address public primaryFactoryAddress;
-    address public secondaryRouterAddress;
-    address public secondaryFactoryAddress;
```

Comment:
Remove concept of primary/secondary router in favor of allowing explicit overrides instead